build.gradle

```diff
@@ -50,11 +50,14 @@ processResources {
 repositories {
 	maven {
 		name "shadowfacts"
-		url "http://mvn.rx14.co.uk/shadowfacts/"
+		url "http://maven.shadowfacts.net/"
 	}
 	maven {
 		url "http://dvs1.progwml6.com/files/maven"
 	}
+	maven {
```

Comment:
This shouldn't be necessary. Remove this and add `jcenter()` to the beginning of the `repositories` block.